packages/atomic/src/index.html

```diff
@@ -73,7 +73,7 @@
               <atomic-sort-criteria caption="mostRecent" criteria="date descending"></atomic-sort-criteria>
             </atomic-sort-dropdown>
           </div>
-
+          <atomic-no-results></atomic-no-results>
```

Comment:
Can this component be placed anywhere inside the search interface? Or does it need to be next to result list to display correctly?

Reply:
Anywere